# Remove -B/usr/bin from linker_bin_path_flag, as it causes a possibly
# incompatible linker from the system to be used.
#
# Fokke Dijkstra, 20190228
# ported to TensorFlow 1.14.0 by Kenneth Hoste (HPC-UGent)
--- tensorflow-1.14.0/third_party/gpus/cuda_configure.bzl.orig	2019-08-15 10:16:55.337496077 +0200
+++ tensorflow-1.14.0/third_party/gpus/cuda_configure.bzl	2019-08-15 10:21:35.527890849 +0200
@@ -1121,7 +1121,7 @@
     if should_download_clang:
         cuda_defines["%{linker_bin_path}"] = ""
     else:
-        cuda_defines["%{linker_bin_path}"] = host_compiler_prefix
+        cuda_defines["%{linker_bin_path}"] = ''
 
     cuda_defines["%{extra_no_canonical_prefixes_flags}"] = ""
     cuda_defines["%{unfiltered_compile_flags}"] = ""
